src/components/shared/nav-bar/desktop-nav/desktop-nav.jsx

```diff
@@ -14,7 +16,10 @@ export default function DesktopNav({ toggleMobileNavView, toggleSearchBar }) {
       >
         <MagnifyingGlass />
       </button>
-      <PrimaryButton href='/mail' classNames={styles.nextLink}>
+      <PrimaryButton
+        href='/mail'
+        classNames={route === '/mail' && styles.joinEmailButtonActive}
```

Comment:
You should build this alternative styling into the PrimaryButton component itself, and that we can easily re-use it in the future. And you can make it configurable using props. 
The goal implementation should be something like:
```
<PrimaryButton href="/mail" dark />
```
(Remeber that `dark` is the same as `dark={true}`